admin/package.json

```diff
@@ -150,5 +156,6 @@
       "<rootDir>/services/data-access"
     ],
     "verbose": false
-  }
-}
+  },
+  "mtc[assets-version]": "2c8bde8dabf32d6b9b64059875461c22"
```

Comment:
why the duplication?

Reply:
thanks for spotting this, it was a leftover from a test I made.
it's removed now.